planex/cmd/createmockconfig.py

```diff
@@ -53,15 +53,18 @@ def load_mock_reference(fname):
     return config_opts
 
 
-def load_yum_repos(repo_names):
+def load_yum_repos(includes, excludes):
     """
     read in the yum repository configuration
     """
     yum_base = yum.YumBase()
-    yum_repos = []
-    for repo_id in repo_names:
-        yum_repos += yum_base.repos.findRepos(repo_id)
-    return yum_repos
+    enablelist = []
```

Comment:
Please use list (or set) comprehensions instead of loops here

Reply:
@euanh Refined. Thanks!